lib/rubocop/cop/style/mutable_constant.rb

```diff
@@ -21,6 +21,9 @@ module Style
       #
       # NOTE: Regexp and Range literals are frozen objects since Ruby 3.0.
       #
+      # NOTE: From Ruby 3.0, this cop allows explicit freezing of interpolated
```

Comment:
Initially, I wanted to link https://rubyreferences.github.io/rubychanges/3.0.html#interpolated-string-literals-are-no-longer-frozen-when--frozen-string-literal-true-is-used but it seems we are not linking to external resources in inline docs very often.